src/valgrind/mod.rs

```diff
@@ -2,10 +2,10 @@
 
 pub mod xml;
 
-use std::fmt;
 use std::net::{SocketAddr, TcpListener};
 use std::process::Command;
 use std::{ffi::OsStr, process::Stdio};
+use std::{fmt, io::Read};
```

Comment:
This formatting change was not intended, but can be kept (or changed back the next time).
```suggestion
use std::fmt;
use std::io::Read;
```